core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.ts

```diff
@@ -113,6 +118,8 @@ export class VisualizationPanelContentComponent implements AfterViewInit, OnDest
     if (this.data?.length < 1) {
       return;
     }
+    this.displayHTML = false;
+    this.displayMap = false;
```

Comment:
as we get more viz types, we may consider making an enum like `vizType` in the future.

Reply:
yes we will decouple all viz operators in the future